doc/use_cases/navigation_with_cartographer.md

```diff
@@ -0,0 +1,44 @@
+
+**Navigation 2 with Cartogtographer**
+
+This document explains how to use Navigation 2 with Cartographer. The following steps show ROS 2 users how to generate occupancy grid maps and use Navigation 2 to move their robot around.
+
+## Before the Tutorial
+
+- Install ROS 2
+- Install Navigation 2
+- Install Cartographer
+- Install your robot packages. 
+- Source setup.bash in your ROS2 and Navigation 2 installation/build directory or workspaces
+- Setting the ROS Domain ID variable is recommended. Ex:
+    - `export ROS_DOMAIN_ID=30`
+
+## Tutotrial Steps
+
+**1- (Robot)**  Bring up the robot-related packages on your robot. 
+Make sure /tf and /odom are being published.
+	- *Ex: Turtlebot 3*
+            
+       - ros2 launch turtlebot3_bringup robot.launch.py
+    
+**2- (Robot or PC)** Bring up Navigation
+This will bring up Navigation 2 without nav2_amcl and nav2_map_server. Cartographer will publish to /map topic and do localization.
```

Comment:
Does this auto-bringup by default now?

Reply:
ah sorry, I meant  does it auto-bringup == lifecycle autostart, so if you run that will everything auto transition into the active state, otherwise that's a step we need to add

Reply:
yes auto-start default value is set to true.